SocialCareCaseViewerApi.Tests/V1/Boundary/Response/CreateWarningNoteResponseTests.cs

```diff
@@ -0,0 +1,30 @@
+using FluentAssertions;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Boundary.Response
+{
+    [TestFixture]
+    public class CreateWarningNoteResponseTests
+    {
+        private CreateWarningNoteResponse _classUnderTest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _classUnderTest = new CreateWarningNoteResponse();
+        }
+
+        [Test]
+        public void ResponseHasWarningNoteId()
```

Comment:
Maybe just semantics but the names of these test methods don't reflect what they are testing.  A possible way of testing an entity property might be `Assert.That(entity, Has.Property("WarningNoteId").InstanceOf(typeof(int)));`

Reply:
I'll flag that with Tuomo, as if we're changing it here it might be best to also change all the boundary and infrastructure tests that have the same format, which could be done as a separate branch